test/peer-store/address-book.spec.js

```diff
@@ -41,21 +41,33 @@ describe('addressBook', () => {
     })
 
     it('throwns invalid parameters error if invalid PeerId is provided', () => {
-      expect(() => {
+      try {
         ab.set('invalid peerId')
-      }).to.throw(ERR_INVALID_PARAMETERS)
+      } catch (err) {
+        expect(err.code).to.equal(ERR_INVALID_PARAMETERS)
+        return
+      }
+      throw new Error('invalid peerId should throw error')
```

Comment:
Any reason for changing these over to try/catches? If you were having errors running this file by itself it's likely due to missing `chai.use(require('chai-as-promised'))` in the require statements.

Reply:
These tests were working correctly before because the error was not correctly required

If I try to use the approach with `chai-as-promised`, I got he error: `TypeError: [Function] is not a thenable.`, which makes sense because this is not a promise.